### PR TITLE
Create new appstore page

### LINKED ISSUE #2492: Build IoT appstore page
Copydoc: https://docs.google.com/document/d/1CKwmPDDtXOIgw-jSZFwzP-qLMc_rS_dUACpHQXlg0fA/edit

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

* Created new /internet-of-things/appstore page
* Added a direct from /iot-appstore

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [new page](http://0.0.0.0:8001/internet-of-things/appstore)
- compare it to the [copy doc](https://docs.google.com/document/d/1CKwmPDDtXOIgw-jSZFwzP-qLMc_rS_dUACpHQXlg0fA/edit#)

## Issue / Card

Fixes #2492

### CODE REVIEW

File: navigation.yaml [hunk 1]
```diff
@@ -158,6 +158,8 @@ iot:
       path: /internet-of-things/robotics
     - title: Gateways
       path: /internet-of-things/gateways
+    - title: App store
+      path: /internet-of-things/appstore
```
Comment (anthonydillon) [inline navigation.yaml]: Shouldn't this URL be `app-store` of its to different words?

File: templates/internet-of-things/appstore.html [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
+  <div class="row u-equal-height">
+    <div class="col-8">
+      <h1>An IoT app store for all your devices</h1>
+      <p>Manufacturers can cut the cost of managing software updates on their devices and take advantage of new revenue opportunities by selling services via their own branded IoT app store &mdash; hosted by Canonical.</p>
+      <ul class="p-inline-list">
+        <li class="p-inline-list__item">
+          <a class="p-button--positive" href="https://www.brighttalk.com/webcast/6793/290087" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'External Link', 'eventAction' : 'Webinar', 'eventLabel' : 'Watch the Path To IoT Revenue webinar', 'eventValue' : undefined });"><span class="p-link--external">Watch the 'Path To IoT Revenue' webinar</span></a>
+        </li>
+        <li class="p-inline-list__item">
+          <a href="/internet-of-things/contact-us?product=appstore" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Internal Link', 'eventAction' : 'Contact us', 'eventLabel' : 'Telcos - Top CTA' });" class="p-button--neutral">Contact us&nbsp;&rsaquo;</a>
```
Comment (anthonydillon) [inline templates/internet-of-things/appstore.html]: Buttons shouldn't have chevrons.

File: templates/internet-of-things/appstore.html [hunk 3]
```diff
@@ -0,0 +1,220 @@
...
+{% block content %}
+
+<section class="p-strip--accent is-deep">
+  <div class="row u-equal-height">
+    <div class="col-8">
+      <h1>An IoT app store for all your devices</h1>
+      <p>Manufacturers can cut the cost of managing software updates on their devices and take advantage of new revenue opportunities by selling services via their own branded IoT app store &mdash; hosted by Canonical.</p>
+      <ul class="p-inline-list">
+        <li class="p-inline-list__item">
+          <a class="p-button--positive" href="https://www.brighttalk.com/webcast/6793/290087" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'External Link', 'eventAction' : 'Webinar', 'eventLabel' : 'Watch the Path To IoT Revenue webinar', 'eventValue' : undefined });"><span class="p-link--external">Watch the 'Path To IoT Revenue' webinar</span></a>
```
Comment (anthonydillon) [inline templates/internet-of-things/appstore.html]: Can you make this sentence case please.

File: templates/internet-of-things/appstore.html [hunk 4]
```diff
@@ -0,0 +1,220 @@
...
+      <img class="hero-image" src="{{ ASSET_SERVER_URL }}1570841c-icon-app-store.svg" alt="">
+    </div>
+  </div>
+</section>
+
+<section class="p-strip is-deep is-bordered">
+  <div class="row p-divider">
+    <div class="col-4 p-divider__block">
+      <h2 class="p-heading-three">Control the user&rsquo;s experience</h2>
+      <p>The option of a private or public store enables you to select which software and which apps are available to your customers, while device access ensures only authenticated users can access your store.</p>
```
Comment (anthonydillon) [inline templates/internet-of-things/appstore.html]: Note: this is a long sentence without breaks.

File: templates/internet-of-things/appstore.html [hunk 5]
```diff
@@ -0,0 +1,220 @@
...
+    <div class="col-4 u-align--center u-vertically-center u-hide--small">
+      <img class="hero-image" src="{{ ASSET_SERVER_URL }}1570841c-icon-app-store.svg" alt="">
+    </div>
+  </div>
+</section>
+
+<section class="p-strip is-deep is-bordered">
+  <div class="row p-divider">
+    <div class="col-4 p-divider__block">
+      <h2 class="p-heading-three">Control the user&rsquo;s experience</h2>
```
Comment (anthonydillon) [inline templates/internet-of-things/appstore.html]: `p-heading--three`

File: templates/internet-of-things/appstore.html [hunk 6]
```diff
@@ -0,0 +1,220 @@
...
+      <p>The option of a private or public store enables you to select which software and which apps are available to your customers, while device access ensures only authenticated users can access your store.</p>
+    </div>
+    <div class="col-4 p-divider__block">
+      <h2 class="p-heading-three">Managing your store is simple</h2>
```
Comment (anthonydillon) [inline templates/internet-of-things/appstore.html]: `p-heading--three`

File: templates/internet-of-things/appstore.html [hunk 7]
```diff
@@ -0,0 +1,220 @@
...
+      <p>Procedures and standards for your store are defined by you, or you could chose to adhere to an existing IT policy within a private store. You can also grant or deny access to selected developers who are authorised to distribute and upload content.</p>
+    </div>
+    <div class="col-4 p-divider__block">
+      <h2 class="p-heading-three">Continued monetisation</h2>
```
Comment (anthonydillon) [inline templates/internet-of-things/appstore.html]: `p-heading--three`

Comment (pmahnke) [reply templates/internet-of-things/appstore.html]: done.

File: templates/internet-of-things/appstore.html (hunk 6 above)
Comment (pmahnke) [reply templates/internet-of-things/appstore.html]: done.

File: templates/internet-of-things/appstore.html (hunk 4 above)
Comment (pmahnke) [reply templates/internet-of-things/appstore.html]: Fixed

File: templates/internet-of-things/appstore.html (hunk 5 above)
Comment (pmahnke) [reply templates/internet-of-things/appstore.html]: done.

File: templates/internet-of-things/appstore.html (hunk 2 above)
Comment (pmahnke) [reply templates/internet-of-things/appstore.html]: Fixed

File: templates/internet-of-things/appstore.html (hunk 3 above)
Comment (pmahnke) [reply templates/internet-of-things/appstore.html]: done, but bad on brighttalk too.

File: navigation.yaml (hunk 1 above)
Comment (anthonydillon) [reply navigation.yaml]: I remember we said we would use `appstore` in URLs and app store in body text.

**Comment by frankiegranato** [comment]
Looks great. Brilliant. 
Just a few small pointers:
- [ ] I like the app icon in the hero section. Lets ask @spencerbygraves to tidy it up a bit for us (ie. remove the white shadow?). 
- [ ] Can we change the colour in the background of the "Features" section please? (just so there is some differentiation from the "Get started" section above)
- [ ] Lets add a heading to the "Features" section (I've made the amendment in the doc)
- [ ] Lets add a heading to the "Resources" section (amendment in the doc)
- [ ] The font style in 'Resources type' headings is incorrect (should be small caps I think)
- [ ] Also in the resources section, the external icons are touching the baseline in the text above. See an example of where this is corrected on: /security
- [ ] Re the Testimonial section: see note in copydoc.
- [ ] Re the 'Used By' section, the last logo is too small - its almost illegible. Can you remedy this at all?

**Comment by frankiegranato** [comment]
@spencerbygraves can you review the use of the pictograms in the 'Enterprise support...' section? I like the idea of using them but, do they look a bit floaty? Maybe the size or positioning needs adjusting.

**Comment by pmahnke** [comment]
@frankiegranato 

> I like the app icon in the hero section. Lets ask @spencerbygraves to tidy it up a bit for us (ie. remove the white shadow?).

I have used the paper background now, I think it looks better... you?

> Also in the resources section, the external icons are touching the baseline in the text above. See an example of where this is corrected on: /security

actually, this is an enhancement for Vanilla... @kwm14 - what do you think?  I think padding-top could be a lot smaller

**Comment by spencerbygraves** [comment]
>@spencerbygraves can you review the use of the pictograms in the 'Enterprise support...' section? I like the idea of using them but, do they look a bit floaty? Maybe the size or positioning needs adjusting.

I think they could be a touch bigger, 90% width rather than 80%.

> I like the app icon in the hero section. Lets ask @spencerbygraves to tidy it up a bit for us (ie. remove the white shadow?).

I don't think it's a problem, let's leave it as designed.

The spacing in the 'used by' section looks a bit big. Looking at other examples we don't seem to split each row into a separate ul, which is adding margin-top.

Thanks

**Comment by frankiegranato** [comment]
Thank you @spencerbygraves 
@pmahnke, can you increase the size of the pictograms in the 'Enterprise support...' as Spencer advises please?
Also, @pmahnke it sounds like the layout of the logos needs addressing. Maybe we should just go back to the tiny-weeny version of the last logo. i didn't realise that imbiggening it would have a knock-on effect on the whole row.

**Comment by frankiegranato** [comment]
@pmahnke I trust you'll address Spencer's two comments (one re 'Enterprise support...' section and one re the logos section)?

From a UX perspective I think everything else is looking great.

**Comment by frankiegranato** [comment]
Opps - I just spotted that Thibaut has requested we change out a resource. Please see copy doc. UX+1 revoked :-(

File: templates/internet-of-things/appstore.html [hunk 8]
```diff
@@ -0,0 +1,228 @@
...
+    <div class="col-4 p-divider__block" id="ubuntu-com_appstore_right">
+      <!-- rtp section start -->
+      <div class='p-heading-icon u-no-margin--bottom'>
+        <div class='p-heading-icon__header u-no-margin--bottom'>
+          <img class='p-heading-icon__img' src='{{ ASSET_SERVER_URL }}5cf7bd32-news-feed-strip-case-study.svg' alt='' style='max-width: 30px;' />
+          <h3 class='p-heading-icon__title p-muted-heading' style='color: #666;'>Case study</h3>
+        </div>
+      </div>
+      <h2 class='p-heading--four'>
+        <a class='p-link--external' href='https://insights.ubuntu.com/2017/12/11/rocket-chat-communication-platform-enables-simplicity-through-snaps/' onclick='dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'product page', 'eventAction' : 'appstore - right', 'eventLabel' : 'Case study - Rocket.chat archives simplicity through snaps', 'eventValue' : '1' });'><span hidden>Read the case study </span>Rocket.chat archives simplicity through snaps</a>
```
Comment (caldav) [inline templates/internet-of-things/appstore.html]: This caught my eye: archives -> achieves

Comment (pmahnke) [reply templates/internet-of-things/appstore.html]: good catch, but it is being removed... sorry

**Comment by pmahnke** [comment]
dealt with logos and updated the resource

**Comment by frankiegranato** [comment]
Looks great.

**Pull request merged**